tests/pytorch/test_datasets_pt.py

```diff
@@ -32,6 +32,8 @@ def test_visiondataset():
         ['IIIT5K', False, [32, 128], 3000, False],
         ['SVT', True, [512, 512], 100, True],
         ['SVT', False, [512, 512], 249, False],
+        ['SVHN', True, [32, 128], 33402, True],
+        ['SVHN', False, [32, 128], 13068, False],
```

Comment:
Is it slow to run as a test?

Reply:
@fg-mindee 
it takes ~ 3.5 min

Reply:
That's too long, we cannot add 3.5min to the CI for a single dataset :sweat_smile:

Reply:
Sorry, circling back on this, I think we need to create a small fake SVHN file at runtime (create something in the same format for the CI)